README.md

```diff
@@ -238,6 +238,12 @@ The filtering can also be applied for pins with `subscribe` attribute.
 
 ## Changelog
 
+### v4.7.0
+
+#### Feature:
+
+* gRPC interface for codec pipeline
```

Comment:
Please also add some info about it in readme (what it supports, link to proto description, etc...)